scaleway/helpers_lb.go

```diff
@@ -207,3 +207,29 @@ func expandLbHCHTTPS(raw interface{}) *lb.HealthCheckHTTPSConfig {
 		Code:   expandInt32Ptr(rawMap["code"]),
 	}
 }
+
+func expandLbCertificateLetsEncrypt(raw interface{}) *lb.CreateCertificateRequestLetsencryptConfig {
+	if raw == nil || len(raw.([]interface{})) != 1 {
+		return nil
+	}
+	rawMap := raw.([]interface{})[0].(map[string]interface{})
+	alternativeNames := rawMap["subject_alternative_name"].([]interface{})
+	config := &lb.CreateCertificateRequestLetsencryptConfig{
+		CommonName:             rawMap["common_name"].(string),
+		SubjectAlternativeName: make([]string, 0, len(alternativeNames)),
+	}
+	for _, alternativeName := range alternativeNames {
+		config.SubjectAlternativeName = append(config.SubjectAlternativeName, alternativeName.(string))
+	}
+	return config
+}
+func expandLbCertificateCustomCertificate(raw interface{}) *lb.CreateCertificateRequestCustomCertificate {
```

Comment:
What do you think about `expandLbCustomCertificate`?

Reply:
OK